tensorrt_llm/serve/openai_disagg_server.py

```
@@ -487,6 +507,36 @@ async def send_completion_request(self, url: str, request: CompletionRequest) ->
+                    destination_ts = time.monotonic()
+                    if response.status == 200:
+                        response = await response.json()
+                        # Compute the steady clock timestamp difference using the NTP clock synchronization algorithm. https://en.wikipedia.org/wiki/Network_Time_Protocol#Clock_synchronization_algorithm
+                        receive_ts = response['receive_ts']
+                        transmit_ts = response['transmit_ts']
+                        delay = (destination_ts - originate_ts) - (transmit_ts - receive_ts)
+                        offset = ((receive_ts - originate_ts) + (transmit_ts - destination_ts)) / 2
+                        return delay, offset
+                    else:
+                        return None, None
+            except Exception:
```

Ruff (0.13.1)

Do not catch blind exception: `Exception`

(BLE001)